clear no results for search list

KK998 opened the pull request:
New no results component
Added the component to itemslist

added new img for the crying emoticon in the new component
added custom styles as bootstrap is not okej.

Commit:
ADD: new component for no search results
ADD: new component to ItemList
ADD: fallback to simple text if no keyword for search

Commit:
FIX: formatting for new files

vanessa-cooper commented:
Got it, will review in a sec’

vanessa-cooper reviewed (changes requested):
Let’s try this again, shall we?

Commit:
FIX: add missing id for no search

Commit:
FIX: styles and id for component

Commit:
try to fix test case?

Commit:
try to get the pr checks
https://discord.com/channels/843966263814389771/1001606610944282624/1002139604310298624
